test_mem/test_mem_genesis2.py

```diff
@@ -1,11 +1,12 @@
 from mem import mem_genesis2
```

Comment:
Can we note somewhere (perhaps in this file or in the README in `mem/` directory) that all of this is related to the `memory_core` module. Since the directory and all of the python assets are prefixed with `mem` there might be confusion that this is for `mem.vp` and not `memory_core.vp`.

Reply:
Done